spec/system/admin/order_cycles/simple_spec.rb

```diff
@@ -258,6 +274,18 @@
         click_button 'Add supplier'
         expect(page).to have_content "Permitted supplier"
 
+        within("tr.supplier-#{supplier_permitted.id}") { click_button 'Add fee' }
+        expect(page).to have_select(
+          "order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_id", minimum: 1
+        )
+        page.find("select#order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_id")
+          .all('option[label]').first.select_option
```

Comment:
I think the `select` helper will work here:
```suggestion
        select "Permitted supplier", from: "order_cycle_incoming_exchange_1_enterprise_fees_0_enterprise_id"
```

Reply:
Thanks. changed